l10n_ch_hr_payroll/data/hr.salary.rule.xml

```diff
@@ -4,616 +4,429 @@ File: hr.salary.rule.xml
 Module: l10n_ch_hr_payroll
 
 Created by sge@open-net.ch
+Updated by lfr@open-net.ch (2017)
 
 Copyright (c) 2015 Open Net Sàrl
 -->
-<openerp>
-    <data noupdate="1">
-        <record id="AM" model="hr.salary.rule">
+<odoo>
+    <data>
```

Comment:
Why are you removing the 'noupdate' ?
If you ar e sure, you can remove also the '<data>'